themes/kross-hugo/layouts/index.html

```diff
@@ -14,6 +14,9 @@ <h2 class="text-white">{{ site.Data.homepage.banner.subText | safeHTML }}</h2>
             <a href="https://marketplace.visualstudio.com/items?itemName=redhat.vscode-kaoto" title="VS Code Extension" class="btn btn-primary btn-cta border mt-4 pt-1 pb-1 mr-2 vscode" target="_blank">
              &nbsp;&nbsp;VS Code Extension
             </a>
+            <a href="https://red.ht/kaoto" title="Try it!" class="btn btn-primary btn-cta border mt-4 pt-1 pb-1 mr-2" target="_blank">
```

Comment:
Could we please add a `rel="nofollow"` here so we don't tell google to leave the site.. probably we should have had this for the VSCode link and any other external one as well

Reply:
sure, I can also add it for VS Code as well

Reply:
I also see that for other external links like Twitter we're using `noopener` as well, I'm adding it also.